packages/widget-v2/src/stories/SwapExecutionPage.stories.tsx

```diff
@@ -57,7 +56,5 @@ type Story = StoryObj<typeof meta>;
 export default meta;
 
 export const SwapExecutionPageExample: Story = {
-  args: {
-    operations: operations as Operation[],
-  },
+  args: {},
```

Comment:
any reason why you move it inside?

Reply:
@codingki yeah I figure that eventually this data will exist in an atom, so we won't be passing it into the component as a prop, so I decided to just move it inside for now (the usage of the component is more similar to how it'll be when we have the data in an atom)